build/common/openwisp/settings.py

```diff
@@ -93,21 +86,20 @@
 
 CHANNEL_LAYERS = {
     'default': {
-        'BACKEND': 'asgi_redis.RedisChannelLayer',
+        'BACKEND': 'channels_redis.core.RedisChannelLayer',
         'CONFIG': {'hosts': [(REDIS_HOST, 6379)]},
-        'ROUTING': 'openwisp_controller.geo.channels.routing.channel_routing',
     },
 }
 
 # Cache
 # https://docs.djangoproject.com/en/2.2/ref/settings/#caches
 
 CACHES = {
-    "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": "redis://"+REDIS_HOST+":6379/1",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
+    'default': {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': 'redis://'+REDIS_HOST+':6379/1',
```

Comment:
why don't you use `format` which is more pythonic?

```python
'redis://{}:6379/1'.format(REDIS_HOST),
```

Are we using python 3.7 or python 3.6?
Because with python 3.7 we can use f strings which are even better:

```python
f'redis://{REDIS_HOST}:6379/1',
```

Reply:
Okay. I'll move all the strings settings to f-string as soon as possible. :smile: